app/views/maintenance/service_unavailable.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1 class="govuk-heading-l govuk-!-width-two-thirds">
+  Sorry, the service is unavailable
+</h1>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <p class="govuk-body">You will be able to use the service later from TIME on DAY.</p>
+    <p class="govuk-body">We saved your answers.</p>
```

Comment:
Here's my current understanding of whether answers get saved or not:
- If maintenance mode gets activated whilst a user is on a page, anything they've done on that page (e.g. making a change to an answer on a log, or submitting a bulk upload) will be lost.
- Otherwise data is saved.

That seems to be the case from my testing and my understanding of what's going on, but I wanted to check here with people more experienced with Rails and this codebase in case there are any edge cases I'm missing.

Assuming I'm right, what do people think would be sensible content here? "We have not saved any changes you made immediately prior to the site undergoing maintenance"?

Reply:
I agree with your understanding on what does and doesn't get saved. 

I think if I was in the middle of filling out a log though and was told that my answers hadn't been saved (including with any immediately prior wording), I'd expect the entire log's worth of answers not to be saved. Although I guess the buttons on each question do say "save and continue". Probably worth asking Susannah about wording.